packages/icons/icons.json

```
@@ -3960,5 +3960,14 @@
         "tags": "circle, cross mark, fail, delete, no, close, remove, not, available, unavailable, wrong, incorrect, cancel",
         "group": "action",
         "codepoint": 62262
+    },
+
+    {
+        "displayName": "Intelligence",
+        "iconName": "intelligence",
+        "tags": "intelligence, intel, observe, watch, capture, humint",
+        "group": "interface",
```

is this really an interface icon? let's throw it in misc for now

```suggestion
        "group": "miscellaneous",
```